content/kotlin/concepts/datetime/terms/periodUntil/periodUntil.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: '.periodUntil()'
+Description: 'returns an object representing the difference between two dates in terms of years, months and days.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **periodUntil()** function is used to calculate the difference between two `LocalDate` or `Instant` instances. This returns an `object` representing the difference between the two dates in terms of years, months and days.
+
+## Syntax
+
+```pseudo
+expect fun LocalDate.periodUntil(other: LocalDate): DatePeriod
+```
+
+`DatePeriod`gives the difference between two instants, decomposed into date and time components where all time components are equals to zero.
+
+## Example
+
+```kotlin
+val startDate = LocalDate.of(2023,11,2)
+val endDate = LocalDate.of(2023,11,3)
+
+val period = startDate.periodUntil(endDate)
+
+println("difference: $period.days day")
```

Comment:
The code is not compiling properly, please check if the code is working or not.

Reply:
Hi @avdhoottt, where to add the prerequisites section? Inside the example or have to make a new section at the end?

Reply:
Use this [guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md) and write a note describing what the user will need in order to run this code. Add the note above the code.